lib/actions/action-send-email.ts

```diff
@@ -81,13 +81,13 @@ const handler: ActionDefinition['handler'] = async (
 		defaultEnvironment.mail.options
 	) {
 		const client = new Mailgun(defaultEnvironment.mail.options);
-		const response = await client.sendEmail({
```

Comment:
This is an axios response object typed as `any` and therefore doesn't match what we expect to be returned from action handlers. `null` is the best option here as we aren't creating/updating contracts etc, we're just attempting to send an email through an external service.